### DISCUSSION TITLE
Is it possible to keep unselected tabs loaded?

### DISCUSSION THREAD

**Discussion by nicogranelli** [mantinedev/mantine#1093] [Q&A] [answered]
I have a couple of tabs with bigs tables on them, and they take some time to render. 

Is it possible to keep them loaded when the active tab changes?

**Comment by rtivital** [comment, marked as answer]
You can use controlled tabs and render content on your side. 

https://codesandbox.io/s/suspicious-cdn-9uldr9?file=/src/App.tsx

**Reply by nicogranelli** [reply]
Thanks!
